lib/takeaway.rb

```diff
@@ -0,0 +1,33 @@
+class Restaurant
+    attr_reader :menu, :order
+
+    def initialize
+        @menu = { 'burger': 5,
+                  'pizza': 6,
+                  'lasagna': 4,
+                  'ice_cream': 2
+                }
+        @order = []
+    end 
+
+    def menu_print
+        @menu.each { |key, value| puts "#{key}: £#{value}" }
```

Comment:
Is there a way you could output the price in correct format £6.00. You could try '%.2f' % value